lightning-persister/src/lib.rs

```diff
@@ -93,28 +109,8 @@ impl FilesystemPersister {
 		path
 	}
 
-	/// Writes the provided `ChannelManager` to the path provided at `FilesystemPersister`
-	/// initialization, within a file called "manager".
-	pub fn persist_manager<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref>(
-		data_dir: String,
-		manager: &ChannelManager<Signer, M, T, K, F, L>
-	) -> Result<(), std::io::Error>
-	where
-		M::Target: chain::Watch<Signer>,
-		T::Target: BroadcasterInterface,
-		K::Target: KeysInterface<Signer=Signer>,
-		F::Target: FeeEstimator,
-		L::Target: Logger,
-	{
-		let path = PathBuf::from(data_dir);
-		util::write_to_file(path, "manager".to_string(), manager)
-	}
-
-	/// Write the provided `NetworkGraph` to the path provided at `FilesystemPersister`
-	/// initialization, within a file called "network_graph"
-	pub fn persist_network_graph(data_dir: String, network_graph: &NetworkGraph) -> Result<(), std::io::Error> {
-		let path = PathBuf::from(data_dir);
-		util::write_to_file(path, "network_graph".to_string(), network_graph)
+	pub fn path_to_monitor_data_str(&self) -> &str {
+		self.path_to_monitor_data().to_str().unwrap()
```

Comment:
Oof. Let's not keep converting PathBuf->str->String->PathBuf->.... quite a waste of allocations.

Reply:
yeah, well if we are okay with building path using "/" then we don't need to do this I guess?  In the current code the method to get monitors path returned a PathBuf but we need a String key for the new trait.  Only way to avoid this is to return a String for the monitors path using "/"

Reply:
Hmm, I guess Winblowz still uses "\".